components/apimgt/org.wso2.carbon.apimgt.keymgt/src/main/java/org/wso2/carbon/apimgt/keymgt/handlers/DefaultKeyValidationHandler.java

```diff
@@ -261,4 +261,10 @@ public boolean validateScopes(TokenValidationContext validationContext) throws A
         }
         return true;
     }
+
+    public boolean validateSubscription(TokenValidationContext tokenValidationContext)
+            throws APIKeyMgtException {
+        //todo validate using inmemory store
+        return true;
```

Comment:
shall we keep the existing subscription validation logic and remove it once the in-memory validation is there

Reply:
sure